<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="53834" started-at="2018-07-25T07:00:54Z" finished-at="2018-07-25T07:01:47Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="1180" started-at="2018-07-25T07:00:54Z" finished-at="2018-07-25T07:00:55Z">
      <class name="com.accenture.test.LoginTest">
        <test-method status="FAIL" signature="LoginInvalid()[pri:0, instance:com.accenture.test.LoginTest@7c30a502]" name="LoginInvalid" duration-ms="47" started-at="2018-07-25T12:30:55Z" finished-at="2018-07-25T12:30:55Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Tested] but found [Tested123]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Tested] but found [Tested123]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at com.accenture.test.LoginTest.LoginInvalid(LoginTest.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginInvalid -->
        <test-method status="PASS" signature="LoginValid()[pri:0, instance:com.accenture.test.LoginTest@7c30a502]" name="LoginValid" duration-ms="1" started-at="2018-07-25T12:30:55Z" finished-at="2018-07-25T12:30:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginValid -->
      </class> <!-- com.accenture.test.LoginTest -->
    </test> <!-- Test -->
    <test name="Test2" duration-ms="51221" started-at="2018-07-25T07:00:56Z" finished-at="2018-07-25T07:01:47Z">
      <class name="com.accenture.test.RegistrationTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.accenture.test.RegistrationTest@668bc3d5]" name="beforeTest" is-config="true" duration-ms="28752" started-at="2018-07-25T12:30:56Z" finished-at="2018-07-25T12:31:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="TestRegistration()[pri:0, instance:com.accenture.test.RegistrationTest@668bc3d5]" name="TestRegistration" duration-ms="22453" started-at="2018-07-25T12:31:25Z" finished-at="2018-07-25T12:31:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestRegistration -->
      </class> <!-- com.accenture.test.RegistrationTest -->
    </test> <!-- Test2 -->
  </suite> <!-- Suite -->
</testng-results>
